Framework/PythonInterface/plugins/algorithms/EnggSaveSinglePeakFitResultsToHDF5.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.api import *
+from mantid.kernel import *
```

Comment:
Is it possible to specify the imports here rather than importing everything ?

Reply:
Just in the case of algorithms I kind of prefer doing `import *` - it keeps the file looking a bit cleaner and makes things easier to fiddle around with in the future - I like how it makes my IDE feel like the Mantid script window. I have yet to run into any naming conflicts. Plus it's [how the docs recommend](https://www.mantidproject.org/Basic_PythonAlgorithm_Structure)